src/main/java/core/basesyntax/garage/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.garage;
+
+public class Truck extends Machine{
```

Comment:
There should not be a redundant empty line after the class declaration. Please remove the empty line to follow proper code formatting guidelines.